src/main/java/org/killbill/billing/plugin/vertex/VertexCalculateTaxApiConfigurationHandler.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Equinix, Inc
```

Comment:
nitty: 2020-2023 for the year for Equinix copyright

Reply:
changed